sdk/attestation/azure-security-attestation/README.md

```diff
@@ -83,88 +83,162 @@ For more information on attestation policies, see [Attestation Policy](https://d
 When an attestation instance is running in "Isolated" mode, the customer who created the instance will have provided
 a policy management certificate at the time the instance is created. All policy modification operations require that the customer sign
 the policy data with one of the existing policy management certificates. The Policy Management Certificate Management APIs enable 
-clients to "roll" the policy management certificates.
+clients to add, remove or enumerate the policy management certificates.
 
 
 ## Examples
 
-* [Attest an SGX enclave](#attest-sgx-enclave)
-* [Get attestation policy](#get-attestation-policy)
+* [Instantiate a synchronous attestation client](#create-a-synchronous-attestation-client)
+* [Instantiate an asynchronous attestation client](#create-an-asynchronous-attestation-client)
```

Comment:
We traditionally do not add async samples to the readme. We rather just share a link to the samples folder.

Reply:
I thought you said I should have snippets for both sync and async behaviors.

Reply:
We add snippets link for both on our async and sync clients. We do not add them to readme.  Majority of our Java developers still consume sync methods so we let readme tailor to that.

Reply:
Ok, updating and removing from readme.